### PR TITLE
feat(test): add unit and widget tests

### PR DESCRIPTION
**Pull request by Mino03947**

### CODE REVIEW

**Review by JaronZ — CHANGES_REQUESTED** [review]
The only thing I saw was how hashCode is calculated. The code looks good other than that.

**Review by JaronZ — APPROVED** [review]
Looks good to me

**Review by altergith — APPROVED** [review]
Ziet er netjes uit, alleen nog wat kleine punten en vraagjes.

File: compairifuel/test/models/access_token_test.dart [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      var expected = const AccessTokenModel(
+          accessToken: '1234567abc',
+          tokenType: 'Bearer',
+          expiresIn: 100000,
+          refreshToken: '1234567abc');
+
+      var result = AccessTokenModel.fromJson(json);
+
+      expect(result, equals(expected));
+      expect(result.hashCode, equals(expected.hashCode));
```
[the remark this answers is not in the document]
Comment (Mino03947) [reply compairifuel/test/models/access_token_test.dart:22]: The difference, lies in the hashcode, the hashcode is an extra check to know if the result is the exact same object. It is also necessary to increase the amount of test coverage. A hashcode can return false, while equals can return true.

I increased readability for you in the changes.

**Review by altergith — APPROVED** [review]
Looks good, well done

**Pull request merged**